src/main/kotlin/day05/Wizard.kt

```
@@ -1,62 +1,62 @@
+        if (10 > mp) {
+            println("마나가 부족합니다")
+        } else {
+            mp -= 10
+            hero.hp += 20
+            println("힐을 시전했습니다, 대상 HP: ${hero.hp}")
+        }
+    }
+
+    fun attack(): Unit {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)